fix build with gcc 10

Define bar_exif_key_count and bar_exif_key_list as extern to avoid the
following build failure with gcc 10 (which defaults to -fno-common):

/home/buildroot/autobuild/instance-2/output-1/host/lib/gcc/arm-buildroot-linux-gnueabihf/10.2.0/../../../../arm-buildroot-linux-gnueabihf/bin/ld: info.o:(.rodata+0x0): multiple definition of `bar_exif_key_count'; bar_exif.o:(.rodata+0x0): first defined here
/home/buildroot/autobuild/instance-2/output-1/host/lib/gcc/arm-buildroot-linux-gnueabihf/10.2.0/../../../../arm-buildroot-linux-gnueabihf/bin/ld: info.o:(.bss+0x0): multiple definition of `bar_exif_key_list'; bar_exif.o:(.data+0x0): first defined here

Fixes:
 - http://autobuild.buildroot.org/results/ab19e23cae603b2aa6e696e4937d106883e78b76

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>

diff -Naur gqview-2.1.5.orig/src/bar_exif.h gqview-2.1.5/src/bar_exif.h
--- gqview-2.1.5.orig/src/bar_exif.h	2020-09-09 20:32:50.506755031 +0200
+++ gqview-2.1.5/src/bar_exif.h	2020-09-09 20:33:30.370755818 +0200
@@ -24,8 +24,8 @@
 
 /* these are exposed for when duplication of the exif bar's text is needed */
 
-const gchar **bar_exif_key_list;
-const gint bar_exif_key_count;
+extern const gchar **bar_exif_key_list;
+extern const gint bar_exif_key_count;
 
 gchar *bar_exif_validate_text(gchar *text);
 
